playground/backend/README.md

```diff
@@ -21,47 +21,81 @@
 
 ## About
 
-Apache Beam is an open-source, unified model for defining parallel processing pipelines for batch and streaming data.
-It provides a portable API layer for building sophisticated data-parallel processing pipelines that may be executed
-across a diversity of execution engines, or runners.
-
-Beam Playground helps facilitate trying out and adopting Apache Beam by providing a very quick way for prospective Beam
-users to see and run examples of Apache Beam pipelines in a web interface that requires no setup.
+Beam Playground helps facilitate trying out and adopting [Apache Beam](https://beam.apache.org/) by providing a very
+quick way for prospective Beam users to see and run examples of Apache Beam pipelines in a web interface that requires
+no setup.
 
 ## Getting Started
 
 See [playground/README.md](../README.md) for details on requirements and setup.
 
 This section describes what is needed to run the backend application.
-- Generating models from proto file
-- Go commands to run/test application locally
+
+- Go commands to run/test the backend locally
+- Set up environment variables to run the backend locally
+- Running the backend via Docker
 
 ### Go commands to run/test application locally
 
-The following command is used to build and serve the backend locally:
+Go to the backend directory:
 
+```shell
+$ cd backend
 ```
-go run ./cmd/server/server.go
+
+The following command is used to build and serve the backend locally:
+
+```shell
+$ go run ./cmd/server/server.go
 ```
 
 Run the following command to generate a release build file:
 
-```
-go build ./cmd/server/server.go
+```shell
+$ go build ./cmd/server/server.go
 ```
 
 Playground tests may be run using this command:
 
-```
-go test ... -v
+```shell
+$ go test ... -v
 ```
 
 The full list of commands can be found [here](https://pkg.go.dev/cmd/go).
 
-## Running the server app
+### Set up environment variables to run the backend locally
+
+These environment variables should be set to run the backend locally:
 
-To run the server using Docker images there are `Docker` files in the `containers` folder for Go and Java languages.
-Each of them processes the corresponding SDK, so the backend with Go SDK will work with Go examples/katas/tests only.
+- `BEAM_SDK` - is the SDK which backend could process (`SDK_GO` / `SDK_JAVA` / `SDK_PYTHON`)
```

Comment:
can we allow the backend to support all SDKs? - we can start with a per-language deployment, but we may one day discover that we want a single backend - so I think it would be great to have a parameter to allow this (e.g. `SDK_ALL`).

Also, should Scio be added?

Reply:
I've added `Scio.
We can make changes to allow the backend to support all SDKs. In that case, we do not need this environment variable at all.

Such a change will affect the container (if we have a unified backend for all SDKs, then this image should contain all dependencies for each SDK and it would take more time to deploy, scale, and start). Backend logic would also change (we have differences for each SDK) and scaling part (we are using different logic to scale each SDK’s instances, but if we combine it we couldn’t).
In other words, it is possible but doesn’t make sense in the current architecture.

We can file a ticket for this if you want so it is documented for some future stages